R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -906,6 +910,24 @@ predicted <- predict(model, df)
 head(predicted)
 ```
 
+#### FP-growth
+
+`spark.fpGrowth` executes FP-growth algorithm to mine frequent itemsets on a `SparkDataFrame`.
+
+* `spark.freqItemsets` method can be used to retrieve a `SparkDataFrame` with the frequent itemsets.
+* `spark.associationRules` returns a `SparkDataFrame` with the association rules.
+
+
+```{r}
+items <- selectExpr(createDataFrame(data.frame(items = c(
+  "s,t,u",
```

Comment:
> something that is not coded in 3 lines ;)

That's for sure :) For now I am more trying to figure out how to present this to make it useful. For ML guide we can safely reuse `data/mllib` but I don't think we can do the same with vignette unless we bring `sample_fpgrowth.txt` as a package data.